tools/snp.sh

```diff
@@ -301,6 +301,36 @@ install_rhel_dependencies() {
   pip install tomli
 }
 
+install_fedora_dependencies() {
+  # Build dependencies
+  sudo dnf install -y git make
+
+  # ACL for setting access to /dev/sev
+  sudo dnf install -y acl
+
+  # qemu dependencies
+  sudo dnf install -y ninja-build
+  sudo dnf install -y gcc
+  sudo dnf install -y glib2 glib2-devel
+  sudo dnf install -y pixman pixman-devel
+  sudo dnf install -y meson
+  sudo dnf install -y libslirp-devel
+  sudo dnf install -y libuuid libuuid-devel
+  sudo dnf install -y python
+
+  # ovmf dependencies
+  install_nasm_from_source
+  sudo dnf install -y acpica-tools zstd rpm-build dwarves perl
+
+  # kernel dependencies
+  sudo dnf install -y flex bison
+  sudo dnf install -y openssl-devel
+  sudo dnf install -y elfutils-libelf-devel # to resovle gelf.h: No such file or directory issue
+
+  # cloud-utils dependency
+  sudo dnf install -y cloud-init
+}
+
```

Comment:
Have you made sure the dependencies are not the same as RHEL ? If so you could use the same function.

Reply:
Yes, RHEL and fedora dependencies differs slightly based on the current test...

I thought separating OS dependencies will be useful  since these dependencies can differ over time based on the active changes in AMDESE/AMDSEV, virtee and so on